src/api/models/iuf/iuf-manifest-schema.yaml

```diff
@@ -437,3 +462,21 @@ properties:
               description: >
                 The key of the object to be created in S3.
 
+$defs:
+  MD5Sum:
+    type: string
+    pattern: "^[0-9a-fA-F]+$"
+  IMSArtifact:
+    type: object
+    required:
+    - path
+    additionalProperties: false
+    properties:
+      path:
+        type: string
+        description: >
+          The relative path to the IMS artifact within its image's directory
+      md5sum:
+        description: >
+          The MD5 checksum of the IMS artifact
```

Comment:
It's probably worth explicitly defining behavior if this is omitted, which I think is to do no verification.

Reply:
I'll add a comment to this effect.